packages/components/src/ListView/Items/Item/Item.component.js

```diff
@@ -23,7 +25,11 @@ function Item(props) {
 	}
 
 	const itemId = id && `checkbox-${id}`;
-	const itemSelector = isSwitchBox ? 'switch checkbox' : 'checkbox';
+	const itemSelector = classNames(
```

Comment:
The `itemSelector` purpose was changed but not its name.
It should be rename to something including a mention of `className`.